ui/raidboss/data/06-ew/raid/p6s.txt

```diff
@@ -1,7 +1,121 @@
 ### P6N: Abyssos: The Sixth Circle (Savage)
-#
+# -ii 7865 786A 7870 7871 7872 7873 7875 7879 787D 787E 7885 7886 7888 7889 7890 7892 79EB 79EC 7A18
 
 hideall "--Reset--"
 hideall "--sync--"
 
 0.0 "--Reset--" sync / 21:........:4000000F:/ window 100000 jump 0
+
+
+0.0 "--sync--" sync /Engage!/ window 0,1
+9.5 "--sync--" sync / 14:[^:]*:Hegemone:7860:/ window 10,5
+14.2 "Hemitheos's Dark IV" sync / 1[56]:[^:]*:Hegemone:7860:/
+28.0 "Synergy" sync / 1[56]:[^:]*:Hegemone:(7887|788A):/
+35.1 "--sync--" sync / 1[56]:[^:]*:Hegemone:784C:/
+42.0 "Aetheric Polyominoid" sync / 1[56]:[^:]*:Hegemone:7866:/ window 15,15
+42.0 "--sync--" sync / 1[56]:[^:]*:Hegemone:7854:/
+48.6 "Unholy Darkness" sync / 1[56]:[^:]*:Hegemone:(786D|786E|7891):/
+54.6 "Polyominous Dark IV" sync / 1[56]:[^:]*:Hegemone:7867:/
```

Comment:
This line is the line cleaves from the floor markers, not the raidwide/aoe, so it probably shouldn't be called the same thing in timeline even if the cast is the same name?

Reply:
The raidwide is `Hemitheos's Dark IV`, right?